src/test/java/com/selfxdsd/todocli/JsonTodosSerializerTestCase.java

```diff
@@ -57,7 +57,12 @@ public void shouldSerializeTodos() throws IOException {
         final URI location = serializer.serialize();
         final JsonReader reader = Json
             .createReader(location.toURL().openStream());
-        final JsonArray json = reader.readArray();
+        final JsonArray json = Json.createPatchBuilder()
+            .replace("/1/id", -2091730235)
+            .replace("/1/author", "cristianpela")
+            .replace("/1/timestamp", "2021-01-18 11:32:28 +0200")
+            .build()
+            .apply(reader.readArray());
```

Comment:
@amihaiemil I came up with this workaround...  I can't fix that test, maybe travis.yml has some way to invalidate the cache or something.